src/utils/common.ts

```diff
@@ -292,7 +293,7 @@ export function formatNumber(
 
     if (isNotDefined(options)) {
         formattingOptions.maximumFractionDigits = Math.abs(value) >= 1000 ? 0 : 2;
-        return new Intl.NumberFormat(navigator.language, formattingOptions).format(value);
+        return new Intl.NumberFormat(undefined, formattingOptions).format(value);
```

Comment:
Shouldn't we get this from the user?

Reply:
Yes, we should, but this case is when user doesn't prove any option